test/space_calculation_test.rb

```diff
@@ -26,17 +23,12 @@ def expect_to_execute(command)
   expect(Yast::SCR).to receive(:Execute).with(SCR_BASH_PATH, command)
 end
 
-def fs_type(name)
-  Storage.const_get(:"FsType_#{name.to_s.upcase}")
-end
-
 def filesystem(size_k: 0, block_size: 4096, type: :ext2, tune_options: "")
-  region = double("Storage::Region", block_size: block_size)
-  part = double("Storage::Partition", name: "/dev/sda1", region: region, size: size_k * 1024)
+  region = double("Y2Storage::Region", block_size: block_size)
+  blk_device = double("Y2Storage::BlkDevice", name: "/dev/sda1", region: region, size: size_k * 1024)
```

Comment:
This looks wrong. `Y2Storage::Region#block_size` and `Y2Storage::BlkDevice#size` return `Y2Storage::DiskSize` objects, not integers. So the mocks are wrong.